### DISCUSSION TITLE
When will other languages supported

### DISCUSSION THREAD

**Discussion by zqqqqz2000** [flet-dev/flet#2547] [Q&A]
### Question

When will other languages like golang/rust be supported

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
At the moment we are focused on Python... and there is really a lot to do.
Support for other languages might only be in the long term future or no more. :(

**Reply by FeodorFitsner** [reply]
...or when there is a developer willing to help with adding new language :)
